docker/dev/Dockerfile

```diff
@@ -1,74 +1,44 @@
-# CodeCompass development dependencies
-FROM ubuntu:bionic
+FROM ubuntu:20.04
 
+# tzdata package is installed implicitly in the following command. This package
+# sets timezone interactively during the installation process. This environment
+# variable prevents this interaction.
+ARG DEBIAN_FRONTEND=noninteractive
+
+# CodeCompass development dependencies.
 RUN set -x && apt-get update -qq \
-    && apt-get -y install \
-    cmake make \
-    curl ctags \
-    default-jdk \
-    gcc-7 gcc-7-plugin-dev \
-    libboost-filesystem-dev \
-    libboost-log-dev \
-    libboost-program-options-dev \
-    libboost-regex-dev \
-    libgit2-dev \
-    libgraphviz-dev \
-    libgtest-dev \
-    libmagic-dev \
-    libsqlite3-dev \
-    libssl1.0-dev \
-    llvm-7 clang-7 llvm-7-dev libclang-7-dev \
-    nodejs-dev node-gyp npm
+  && apt-get -y install \
+  cmake make \
+  default-jdk \
+  gcc-7 gcc-7-plugin-dev \
+  libboost-filesystem-dev \
+  libboost-log-dev \
+  libboost-program-options-dev \
+  libboost-regex-dev \
+  libgit2-dev \
+  libgraphviz-dev \
+  libgtest-dev \
+  libmagic-dev \
+  libsqlite3-dev \
+  libssl-dev \
+  llvm-7 clang-7 llvm-7-dev libclang-7-dev \
+  npm \
+  thrift-compiler libthrift-dev \
+  odb
```

Comment:
One minor nit w.r.t. image size: simply installing `odb` will install **all** odb connectors, even mysql and postgres and such, something we're not using inside the image.

Reply:
This comment is still valid.